portability-worker/src/main/resources/settings/common.yaml

```diff
@@ -1,2 +1,11 @@
 env: LOCAL
```

Comment:
Isnt this file copied over by the build script?

Reply:
Let me add a .gitignore

Reply:
This file still shows up in the diff of the request